stores/utxo/aerospike/cleanup/cleanup_service.go

```diff
@@ -351,7 +353,13 @@ func (s *Service) processCleanupJob(job *cleanup.Job, workerID int) {
 
 	// Create a query statement
 	stmt := aerospike.NewStatement(s.namespace, s.set)
-	stmt.BinNames = []string{fields.TxID.String(), fields.DeleteAtHeight.String(), fields.Inputs.String(), fields.External.String()}
+
+	// Add Outputs field if defensive cleanup is enabled
+	binNames := []string{fields.TxID.String(), fields.DeleteAtHeight.String(), fields.Inputs.String(), fields.External.String()}
```

Comment:
Why not just assign directly to "stmt.BinNames"? This extra variable is just another memory alloc that is not needed.